.github/workflows/trailing_commas.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+# This is an pretiffy script.
+# It runs on lib and test directory to find files
+# which doesn't satisfy trailing comma.
+
+import os
+import sys
+import argparse
+
+# Function through parse the filesof the directory
+
+
+def arg_parser_resolver():
+    parser = argparse.ArgumentParser(
+        description='for parsing across the directory')
+    parser.add_argument('--dir', type=str, required=False, default=os.getcwd(),
+                        help='directory-location where files are present')
+    return parser.parse_args()
+
+# patterm matching and adding trailing commas at the required syntax places
+
+
+def syntax_matcher(root: str, files: list):
+
+    for name in files:
+
+        file_location = os.path.join(root, name)
+        file_count = 0
+        data = []
+        g = open(file_location, 'r')
+        data = g.readlines()
+        f = open(file_location, 'w')
+        for index in range(0, len(data)):
+            if ') {' and not '() {' in data[index]:
```

Comment:
The number of elif statement is too much. The code is not generalized efficiently. As this script is automated with GitHub workflow, I think in future this will cause the problem and can lead to more elif cases

Reply:
Yup, I agree there are too many elif. It was necessary and I couldn't find a way to prevent it. Most of the cases will be covered with the present elifs and we have to edit the script for further generalisation

Reply:
But this will cause problems in future PRs. This PR is integrated with the `pull-request.yml` workflow. So if there is not a generalized solution then in future some contributor will push the PR and workflow can cause error